### PR TITLE
Random bits and pieces 5

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

[Update some icons to behave better](https://github.com/webkom/lego-webapp/commit/2a5b580deed3c16fd4d339173510b4e96a40580c) 

They will now act as buttons, so you can tab on them.

---

[Give icons and buttons the same "active" look](https://github.com/webkom/lego-webapp/commit/30e43adfe94702519791c268f321258b1e466fc4) 

The translate(-1px) felt less modern and the icons did not have any
active styling.

---

[Handle copy button timeouts better](https://github.com/webkom/lego-webapp/commit/1c8c866791bc86a962fdb9ee9c5522cd0e5f56c0) 

Less glitchy by clearing out old timeouts when creating new ones.

---

[Fix random bits and pieces, again](https://github.com/webkom/lego-webapp/commit/98de2dfa66cbdd342b933c72ac98845b8ad912b4) 

* Remove some use of italic fonts
* Fix glitchy render on footer
* Remove some minor bad styling

# Result

Less glitchy:

https://github.com/webkom/lego-webapp/assets/69514187/49976684-96bd-423d-9cb4-74e6bd920794

You can now open by tabbing, and the "delete row" has a danger look:

https://github.com/webkom/lego-webapp/assets/69514187/9dfd9d8e-9490-4b70-8efa-d2dea61f64bb

# Testing

- [x] I have thoroughly tested my changes.

Not really any breaking changes, but button and icon changes were tested on both dark and light theme.

### CODE REVIEW

File: app/routes/overview/components/LatestReadme.css [hunk 1]
```diff
@@ -25,3 +24,11 @@
...
   }
 }
+
+.expand {
+  transition: transform var(--easing-medium);
+}
+
+.rotate {
+  transform: rotate(90deg);
+}
```
Comment (ollfkaih) [inline app/routes/overview/components/LatestReadme.css:34]: In my head, 270 degreees would make more sense (going from ➡️ to ⬆️) or even starting at ⬇️ and changing it to 270.

**Review by ollfkaih — APPROVED** [review]
Looks good like usual, just some nitpicking

File: app/components/Upload/ImageUpload.tsx [hunk 2]
```diff
@@ -93,6 +98,9 @@ const UploadArea = ({ multiple, onDrop, image, accept }: UploadAreaProps) => {
       ),
     [isFocused, isDragAccept, isDragReject]
   );
+
+  const word = multiple ? 'bildene' : 'bildet';
```
[the remark this answers is not in the document]
Comment (ivarnakken) [reply app/components/Upload/ImageUpload.tsx:102]: This is actually in master, I just rebased wrong heh

File: app/routes/overview/components/LatestReadme.css (hunk 1 above)
Comment (ivarnakken) [reply app/routes/overview/components/LatestReadme.css:34]: In my head, 90 degrees makes more sense when you're actually moving 90 degrees

Comment (ollfkaih) [reply app/routes/overview/components/LatestReadme.css:34]: ofc, but having the icon to collapse the section look like this ⬇️ seems off

Comment (ivarnakken) [reply app/routes/overview/components/LatestReadme.css:34]: No it looks like ➡️, and then after it is triggered (and rotated) it looks like ⬇️

Comment (ollfkaih) [reply app/routes/overview/components/LatestReadme.css:35]: Yes, the point was having it point down to collapse it (upwards) seems off.

Comment (ivarnakken) [reply app/routes/overview/components/LatestReadme.css:35]: I'm not sure if I'm following you. You mean it seems off that the arrow is pointing down when the content is expanded? Does it not make sense for the arrow to "follow"? I feel like that's the norm

Comment (ollfkaih) [reply app/routes/overview/components/LatestReadme.css:35]: So i found a few faq pages which have these kind of icons. [Instagram](https://help.instagram.com/280908123309761), [Helsenorge](https://www.helsenorge.no/koronavirus) and [SAS](https://www.sas.no/hjelp-og-kontakt/ofte-stilte-sporsmal/?faq-category=baggage&faq-search=) all behave like I would expect in my mind - the SAS one animates exactly like i meant. I did also find a [Microsoft FAQ](https://learn.microsoft.com/en-us/answers/support/qna-faq) where the sidebar has your style of button, so I guess there is a precedent for that too. Also, this is not worth spending more time on and i think your approach looks just fine, so :shipit:

Comment (ivarnakken) [reply app/routes/overview/components/LatestReadme.css:35]: The three examples you refer to all rotate 180 degrees. This one only rotates 90 degrees, like your last example, hence their resembles. Both are, in my opinion, valid designs, and there seems to be [no right way](https://ux.stackexchange.com/questions/5735/which-way-should-arrows-point-in-a-collapsible-accordion-left-down-down-up).

**Pull request merged**